Dockerfile

```diff
@@ -1,7 +1,8 @@
-ARG GO_VERSION=1.20
-ARG LEAP_VERSION=15.5
+ARG GOLANG_IMAGE
+ARG BUILD_OS_DISTRI
+ARG BUILD_OS_VERSION
 
-FROM golang:${GO_VERSION}-alpine as elemental-bin
```

Comment:
Right now the workflow for building multi-platform images includes `make PLATFORM=linux/amd64,linux/arm64 ... build`. Would probably need to do something akin to https://docs.docker.com/build/building/multi-platform/#cross-compilation to get multi-platform working with this change.

Reply:
I will have a look. The main issue is that RISC-V is not always supported (no official Golang image yet for example).

Reply:
I modified and split the build: now we can compile the elemental-cli binary on all architecture using cross-compiling.

There is now two dockerfile: one for the binary and another for the toolkit.